scripts/player.gd

```diff
@@ -1,6 +1,24 @@
 @tool
 extends CharacterBody2D
 
+const _PLAYER_ACTIONS = {
+	Global.Player.LEFT:
+	{
+		"jump": "player_2_up",
+		"left": "player_2_left",
+		"right": "player_2_right",
+	},
+	Global.Player.RIGHT:
+	{
+		"jump": "ui_up",
+		"left": "ui_left",
+		"right": "ui_right",
```

Comment:
If we're defining player 2 actions in the input map, we should define player 1 actions in the input map as well instead of using the built in ui actions.

Reply:
Yes, let's do that. I was only copying Pong for consistency, but I can update both. The good thing about builtin actions like "ui_up" is that they are already mapped for joysticks. I will see if I can do the same for player 1 / player 2.